updated README for run bat for windows (#46)
openMF/fineract-pentaho: README.md

    curl --insecure --location --request GET 'https://localhost:8443/fineract-provider/api/v1/runreports/Expected%20Payments%20By%20Date%20-%20Formatted?R_endDate=2013-04-30&R_loanOfficerId=-1&R_officeId=1&R_startDate=2013-04-16&output-type=PDF&R_officeId=1' --header 'Fineract-Platform-TenantId: default' --header 'Authorization: Basic bWlmb3M6cGFzc3dvcmQ='

The API call (above) will fail on the server (see log) due to an intern error (somehow
the SQL query in that particular report is currently actually broken), but this illustrates
that the integration of Pentaho as a Fineract Plugin basically works.

> ([FINERACT-1176](https://issues.apache.org/jira/browse/FINERACT-1176) tracks improving API response.)

Reviewer: If this issue is still open, how does the API call works? If it is fixed, maybe we can mark this issue as resolved.

Author: https://issues.apache.org/jira/browse/FINERACT-1176 - i have added some comment, and i would wish @vorburger  to close or advise about the issue

This project is currently only tested against the very latest and greatest bleeding edge Fineract `develop` branch. It works like a charm after ([PR #1671](https://github.com/apache/fineract/pull/1671)). Building and using it against older versions may not work. You might want to cherry pick the changes made in the above PR

    git clone https://github.com/apache/fineract.git
    cd fineract && ./gradlew bootJar && cd ..

    git clone https://github.com/vorburger/fineract-pentaho.git
    cd fineract-pentaho && ./gradlew -x test distZip && cd ..

    mkdir -p ~/.mifosx/pentahoReports/
    cp ./fineract-pentaho/pentahoReports/* ~/.mifosx/pentahoReports/

    ./fineract-pentaho/run.bat

> To test this you can any RESTClient tool with the following parameters

Reviewer: ```suggestion
To test this you can any REST Client tool (like Postman) with the following parameters
```

Method : GET

> url    : https://localhost:8443/fineract-provider/api/v1/runreports/Expected%20Payments%20By%20Date%20-%20Formatted?R_endDate=2013-04-30&R_loanOfficerId=-1&R_officeId=1&R_startDate=2013-04-16&output-type=PDF&R_officeId=1

Reviewer: ```suggestion
URL : https://localhost:8443/fineract-provider/api/v1/runreports/Expected%20Payments%20By%20Date%20-%20Formatted?R_endDate=2013-04-30&R_loanOfficerId=-1&R_officeId=1&R_startDate=2013-04-16&output-type=PDF&R_officeId=1
```

> HEADERS

Reviewer: ```suggestion
**HEADERS**
```

Fineract-Platform-TenantId: default
Content-Type: application/json

Authorization: Basic bWlmb3M6cGFzc3dvcmQ=    i.e. (username = mifos & password = password)

If the API call (above) [fails with](https://issues.apache.org/jira/browse/FINERACT-1173) 
_`"There is no ReportingProcessService registered in the ReportingProcessServiceProvider for this report type: Pentaho"`_, 
then this Fineract Pentaho Plugin has not been correctly registered & loaded by Apache Fineract.

> ## WHAT BOTH SCRIPTS DO IN BASIC TERMS

Reviewer: Please no caps.

Author: changes effected

> Both scripts (windows and Linux alike) basically just creates the following directory structure:

Reviewer: ```suggestion
Both scripts (Windows and Linux alike) basically just creates the following directory structure:
```
